app/src/main/java/com/codesoom/assignment/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.codesoom.assignment.exception;
+
+public class ProductNotFoundException extends RuntimeException {
+    public ProductNotFoundException(Long id) {
+        super("There is no product number" + id);
```

Comment:
이렇게 하면 `number`와 `id` 가 붙어서 출력될 것입니다.
```suggestion
        super("There is no product number " + id);
```

Reply:
띄어쓰기확인을 못했네요 감사합니다